recipe/meta.yaml

```diff
@@ -20,13 +20,13 @@ requirements:
   host:
     - python >=3.7
     - pip
-    - flit-core
+    - flit-core >=3.4
   run:
     - python >=3.7
     - colorama
     - importlib-metadata >=0.22
     - packaging >=19.0
-    - pyproject-hooks
+    - pyproject_hooks
```

Comment:
Oh, wow, annoying that this is an underscore when most of the packages use dash. Include this one, flit-core, and importlib-metadata. Just to name ones on this page. :)

Reply:
The problem is in PyPI. They accepted both and folks outside of the Python/PyPI world must choose one. Some conda-forge packages do "name squashing" and use multiple outputs to create both versions with `_` and `-`.

Reply:
Yes, but we could be consistent, at least. The others above all use dashes, and that's what I've used for boost-histogram and others. Using an underscore on a new package is a bit of a papercut, I think. Though not that many projects will likely need it, so it's not a big deal.

Reply:
Who is we? conda-forge? Some authors are publishing their packages here and they choose how they name their stuff. If we want consistency with PyPI the only thing we can do is to "squat" the alternative name to reserve it. Otherwise package publishers will do what package publishers do.